### ISSUE TITLE
[BUG] Playlist cover is the song cover but song is not present in the playlist

### ISSUE THREAD

**Issue by ElionBlitz** [Moosync/moosync-app#257]
**Describe the bug**
When I navigate to playlist tab I can see the album cover of the song X (let's call the song X for convinience) because X is the first song added to that playlist. But in the playlist I cannot find the song X nor can I add it to that playlist, it's invisible as if it's there but it doesn't show or play.

**To Reproduce**
I couldn't reproduce the error unfortunately.

**Expected behavior**
Adding a song to the playlist makes it stay in that playlist.

**Screenshots**
In the Additional Context field

- OS: Windows
- Version 10 Pro

**Additional context**
Here is an example:
The album cover is Metal: Hellsinger
![image_2023-07-20_180552930](https://github.com/Moosync/Moosync/assets/71626536/f30284e4-070b-4c57-a87e-68a362b1acdd)

But inside the playlist it is not there, not visible nor counted for on the song counter
![image_2023-07-20_180636024](https://github.com/Moosync/Moosync/assets/71626536/a5ae5f74-02e9-4b2b-baa7-34f857e5e4c5)

Please let me know if you need further context or clarification I'll be happy to provide it.

**Comment by Ovenoboyo** [comment]
Could you try adding the song again and if it fails, could you check the logs? It seems to me that there is a failure when trying to add that song to the playlist but the cover image is updated. 

Logs can be found in ```settings > logs``` or ```%appdata%/moosync/logs```

**Comment by ElionBlitz** [comment]
![image_2023-07-21_113017677](https://github.com/Moosync/Moosync/assets/71626536/d7d71b45-4e61-4c66-a109-6e4bbeacf72c)
Nothing seems out of the ordinary.

After clicking "Add to Playlist" the "Song added to playlist" didn't pop up. I tried adding other songs to the playlist but nope, either local or youtube music it doesn't matter, it refuses to add anything to the playlists. I'll do an uninstall and re-install, will keep you updated if the issue persists.

**Comment by ElionBlitz** [comment]
Yeah it happened again, all by itself from what it looks like it just decides to lock the playlist... The logs are not showing anything either... And it looks like it deleted some songs off of the playlist too.

![image_2023-07-21_171029100](https://github.com/Moosync/Moosync/assets/71626536/a3d8118a-4bcd-4880-80b1-21568fa5f4d8)
This playlist had 1 song in it and it seems that it just.... dissappeared.
![image_2023-07-21_171140520](https://github.com/Moosync/Moosync/assets/71626536/4b12c7a9-0d5b-46fe-89ef-2a25de54c055)
